### PR TITLE
replace "Clear Filters" with "Cancel"

### PR DESCRIPTION
**Pull request by zbtirrell**
See feedback doc

### CODE REVIEW

File: src/admin-views/aggregator/origins/ics.php [hunk 1]
```diff
@@ -42,8 +42,8 @@ class="tribe-ea-field tribe-ea-media_button tribe-dependent button button-second
 		<button type="submit" class="button button-primary tribe-preview">
 			<?php esc_html_e( 'Preview', 'the-events-calendar' ); ?>
 		</button>
-		<button type="button" class="button tribe-clear-filters">
-			<?php esc_html_e( 'Clear Filters', 'the-events-calendar' ); ?>
+		<button type="cancel" class="button tribe-cancel">
```
Comment (bordoni) [inline src/admin-views/aggregator/origins/ics.php]: Question; Is `cancel` a type for `<button>` I've never seen it. + There isn't any JS associated with the class?

Comment (zbtirrell) [inline src/admin-views/aggregator/origins/ics.php]: I matched this: https://github.com/moderntribe/the-events-calendar/blob/develop/src/admin-views/aggregator/tabs/import-form.php#L177

Behavior is identical to that button.

Comment (borkweb) [inline src/admin-views/aggregator/origins/ics.php]: Ha! I derped that and did `type="cancel" when I should have done `type="reset"`.

**Comment by borkweb** [comment]
🐕 :dash:

**Pull request merged**